daemon/logger/jsonfilelog/jsonfilelog_test.go

```diff
@@ -107,7 +108,10 @@ func BenchmarkJSONFileLoggerLog(b *testing.B) {
 		ContainerID: "a7317399f3f857173c6179d44823594f8294678dea9999662e5c625b5a1c7657",
 		LogPath:     tmp.Join("container.log"),
 		Config: map[string]string{
-			"labels": "first,second",
+			"labels":   "first,second",
+			"max-file": "10",
```

Comment:
IIUC, the intent of this logging driver is to be an "optimised, general-purpose logging driver" (which could become the default).

The storage mechanisms for this driver (currently, file-based, file-format-x) is an implementation detail. Because of that, should we abstract these options? i.e., have options for:

- "maximum amount of logs to keep" (the driver can decide to use multiple files to optimize)
- "maximum age of logs to keep" (in light of GDPR https://github.com/moby/moby/issues/36853)
- compression; good one: is there a significant overhead? are there situations where you _want_ to disable this?